params/config.go

```diff
@@ -359,6 +359,7 @@ type ChainConfig struct {
 	RegolithTime *uint64  `json:"regolithTime,omitempty"` // Regolith switch time (nil = no fork, 0 = already on optimism regolith)
 	CanyonTime   *uint64  `json:"canyonTime,omitempty"`   // Canyon switch time (nil = no fork, 0 = already on optimism canyon)
 	// Delta: the Delta upgrade does not affect the execution-layer, and is thus not configurable in the chain config.
+	EcotoneTime *uint64 `json:"ecotoneTime,omitempty"` // Ecotone switch time (nil = no fork, 0 = already on optimism ecotone)
```

Comment:
We should also add an Ecotone override flag + pull in the time from the superchain registry in params/superchain.go

Reply:
thx will look into this. do the superchain part after merging [this](https://github.com/ethereum-optimism/superchain-registry/pull/46) pr?

Reply:
done